lib/counterstrike.ex

```diff
@@ -80,8 +83,14 @@ defmodule PropCheck.CounterStrike do
   # storing new counter examples.
   @spec load_existing_counter_examples(%{mfa => any}, :dets.tab_name()) :: %{mfa => any}
   defp load_existing_counter_examples(ce, dets) do
-    new_ce = :dets.foldl(fn {mfa, example}, ces ->
-      Map.put_new(ces, mfa, example) end, ce, dets)
+    new_ce =
```

Comment:
while the original source is also not formatted too nicely, but this is quite lengthy, but consistent.

Reply:
Yes, the formatter sometimes produces ugly code. Sometimes, that actually hints to refactor the code a little bit, which can result in better readability. For this example, maybe something like this would be nice:

```elixir
  defp load_existing_counter_examples(ce, dets) do
    new_ce = :dets.foldl(&add_counterexample/1, ce, dets)

    :ok = :dets.delete_all_objects(dets)
    :ok = :dets.sync(dets)
    new_ce
  end

  defp add_counterexample({mfa, example}, ces) do
    Map.put_new(ces, mfa, example)
  end
```